src/Admin/SiteHealth.php

```diff
@@ -334,6 +378,51 @@ public function get_serve_all_templates() {
 		return AMP_Options_Manager::get_option( 'all_templates_supported' ) ? 'true' : 'false';
 	}
 
+	/**
+	 * Gets whether the transient caching of stylesheets was disabled.
+	 *
+	 * @return string Whether the transient caching of stylesheets was disabled.
+	 */
+	private function get_css_transient_caching_disabled() {
+		if ( wp_using_ext_object_cache() ) {
+			return 'n/a';
+		}
+
+		$disabled = AMP_Options_Manager::get_option( Option::DISABLE_CSS_TRANSIENT_CACHING, false );
+
+		return $disabled ? 'true' : 'false';
+	}
+
+	/**
+	 * Gets the threshold being used to when monitoring the transient caching of stylesheets.
+	 *
+	 * @return string Threshold for the transient caching of stylesheets.
+	 */
+	private function get_css_transient_caching_threshold() {
+		/** This filter is documented in src/BackgroundTask/MonitorCssTransientCaching.php */
+		$threshold = (float) apply_filters(
+			'amp_css_transient_monitoring_threshold',
+			MonitorCssTransientCaching::DEFAULT_THRESHOLD
+		);
+
+		return "{$threshold} transients per day";
```

Comment:
Should this be translated?

Reply:
I don't think so. It's part of the Site Health info export.